docker-compose/testnet/docker-compose.yml

```diff
@@ -25,7 +26,7 @@ services:
     environment:
       ELECTRS_NETWORK: "testnet"
       ELECTRS_DAEMON_RPC_ADDR: "bitcoind-testnet:18332"
-      ELECTRS_ELECTRUM_RPC_ADDR: "electrs-testnet:60001"
+      ELECTRS_ELECTRUM_RPC_ADDR: "0.0.0.0:60001"
```

Comment:
Why we change this?

Reply:
I usually make services running in docker listening on `0.0.0.0` otherwise remote calls would fail in most of the cases.

so I've assumed this applied to electrs too, but I've just checked and it also works with `electrs-testnet`, so I'll revert this change. sorry for the mistake

Reply:
I really like to stick to one way of doing this + it works well with limiting connectivity just to specific docker-compose network (since there could be many other bitcoind etc images running in parallel)

Reply:
using `0.0.0.0` is, in my opinion, the best way to stick to if you are searching uniformity. this would not cause any problems even with more instances of bitcoind running in the same network

Reply:
But I am right that it will cause issues with many bitcoind mainnet containers running on the same machine?

Reply:
A docker container represents a machine, many different containers defined in a compose file represent different machines connected to a common private network. So they can all bind a service to the same IP and port (e.g. `0.0.0.0:18332`).

Then if you [expose ports on the machine running the docker containers](https://docs.docker.com/compose/compose-file/#ports) you'll need to expose them on different ports (`18332:18332` , `18333:18332`, `18334:18332`, and so on) because they're all going to listen on the same host address (typically `localhost`).

If you just [expose ports on the common private network](https://docs.docker.com/compose/compose-file/#expose) they can all expose `18332` because, as said before, each container is a different machine